CameraOverlay/OrchestratorClass.py

```diff
@@ -0,0 +1,79 @@
+"Orchestrator Script That Controls The Camera System"
+
+#imports
+import json
+import argparse
+import time
+import paho.mqtt.client as mqtt
+import config
+
+#Get arguments passed into Python script
+
+def get_args(argv=None):
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--host", type=str,
+                        default="192.168.100.100",
+                        help="ip address of the broker")
+    return parser.parse_args(argv)
+
+#Orchestrator Class
+
+class Orchestrator():
+
+    def __init__(self, brokerIP , port = 1883):
+
+        self.BROKER_IP = brokerIP
```

Comment:
In python, we use `snake_case` for member variables and function parameters, i.e. `broker_ip`. (For global variables, which `BROKER_IP` used to be, we use all upper case)